test-app/package.json

```diff
@@ -33,6 +33,7 @@
   "devDependencies": {
     "@babel/core": "^7.20.7",
     "@babel/eslint-parser": "^7.19.1",
+    "@ember/legacy-built-in-components": "0.4.1",
```

Comment:
Why is it needed?

Reply:
I run into the issue running `test-app` and after reading up on `ember-engines` issues i found that the host app requires `@ember/legacy-built-in-components` to run correctly. It looks like it fixes the issues.

Reply:
@SergeAstapov do you have any thoughts on that? I might be a little bit outdated of engines context

Reply:
@villander yes, `@ember/legacy-built-in-component` is needed for Ember.js v4 compatibility due to `<LinkTo />` being internal and non extendable.
this is unfortunately the solution we settled on until we can advance and allow ember-engines do this kind of thing properly, which we had discussions about last year but stalled